common/app/com/commercetools/sunrise/common/template/engine/handlebars/PlayJavaFormResolver.java

```diff
@@ -33,7 +45,7 @@ public Object resolve(final Object context) {
         return Collections.emptySet();
     }
 
-    private Object resolveImpl(@Nonnull final Form<?> form, final String name) {
+    protected Object resolveImpl(@Nonnull final Form<?> form, final String name) {
```

Comment:
I can make some methods private, but I guess your problem is opening this class in general. So we need to enable people to treat errors as they want. I can think of two ways for that: an error formatter class, which is the one you should modify and not this one, or simply opening only certain methods for this purpose.